README.md

```diff
@@ -1,4 +1,8 @@
 # cllm
+[![](https://dcbadge.vercel.app/api/server/c4Jm3r8pBd?style=flat&compact=true)](https://discord.gg/c4Jm3r8pBd)
```

Comment:
<img width="401" alt="image" src="https://github.com/dev-backpack/cllm/assets/52884648/058cd1a0-35b6-48ff-ae05-5d750440ea2f">

Discord badge is not shown!

Reply:
Thank you for your review, and I will check and correct this issue.